### ISSUE TITLE
Problem running with Azure DevOps Server

### ISSUE THREAD

**Issue by rarkins** [renovatebot/renovate#6210]
**What Renovate type are you using?**

Azure DevOps Server

**Describe the bug**

Renovate fails to start, with "Failed to find api location for area" error.

**Did you see anything helpful in debug logs?**

```
renovate --platform=azure --endpoint=https://my.azdodomain.net --token
=*********************************** VulnerableApps/VulnerablePhpApp

ERROR: Repository has unknown error (repository=VulnerableApps/VulnerablePhpApp)
       "err": {
         "message": "Failed to find api location for area: git id: 225f7195-f9c7-4d14-ab28-a83f7ff77e1f",
         "stack": "Error: Failed to find api location for area: git id: 225f7195-f9c7-4d14-ab28-a83f7ff77e1f\n    at /home/trv-azdevops-ubu
ntu18-vm/.npm-global/lib/node_modules/renovate/node_modules/azure-devops-node-api/VsoClient.js:85:23\n    at processTicksAndRejections (int
ernal/process/task_queues.js:97:5)"
       }
```

**To Reproduce**

TBD

**Comment by rarkins** [comment]
@JamieMagee @jycouet do you have any idea what could be going wrong here?

**Comment by JamieMagee** [comment]
Yes, it looks like the version of Azure DevOps Server (or maybe TFS?) that they're using doesn't support one of the APIs that Renovate is using.

Can you get any more information about the specific version of Azure DevOps Server. You can find this under Help -> About.

**Comment by rarkins** [comment]
@gardebring are you able to answer?

Related: https://github.com/jycouet/VSTSExtensions/issues/31#issuecomment-626971740

**Comment by gardebring** [comment]
Hi. The issue in fact seems to be the rights on the "Project collection build service" which was lacking the rights to create branches and Contribute to Pull Request, as spotted by @jycouet here: 
https://github.com/jycouet/VSTSExtensions/issues/31#issuecomment-627226761
All other users had the required rights, including the user I created the token from, but this specific user did not have the rights which seems to have been the issue.
The script now runs, and I do get the onboarding PR.
Interestingly enough renovate however now tells me there are no packages to update, although I know for a fact there is. I tried on a .NET framework project and renovate creates the onboarding PR and then tells me no packages needs updating, while Visual Studio correctly gives me a list of 48 packages to update.
